wp-content/themes/cpipr/lcdm-single.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Template Name: Lcdm single
+ * Template Post Type: post
+ * Description: Shows the post with formatting for custom layout display?
+ */
+
+    // Load up our meta data and whatnot
+    the_post();
+?>
+
+<!DOCTYPE html>
```

Comment:
@jmoreira-linkux is there a specific reason you're copying over the Largo head code here?